install-dev/data/db_structure.sql

```diff
@@ -1696,7 +1697,8 @@ CREATE TABLE IF NOT EXISTS `PREFIX_product_shop` (
   `active` tinyint(1) unsigned NOT NULL DEFAULT '0',
   `redirect_type` ENUM(
     '', '404', '410', '301-product', '302-product',
-    '301-category', '302-category'
+    '301-category', '302-category', '200-displayed', 
+    '404-displayed', '410-displayed', 'default'
```

Comment:
where do we usually put the new configuration values? I think the "PS_PRODUCT_REDIRECTION_DEFAULT" configuration should be saved somewhere with the same value that was default before (404 i think?)

Reply:
I don't understand the question. 😄

This is configuration per single product. If product is using `default` as `redirect_type`, prestashop will check `PS_PRODUCT_REDIRECTION_DEFAULT` stored in `ps_configuration`.

Reply:
Sorry for bad formulation of the questuion. I found where the default configs are installed.
So,
I think you should add the new `PS_PRODUCT_REDIRECTION_DEFAULT` into
`install-dev/data/xml/configuration.xml`
so that the new config value exists after installation by default.

Reply:
Aaaah, now I understand. :-) Forgot it, will do!